sound/audioclass.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+from collections import deque
+import rospy
+import math
+import time
+import sys
+sys.path.append('../../ros-behavior-scripting')
+sys.path.append('../../../vision')
+from std_msgs.msg import Float32
+from face_track.netcat import netcat
+from audiosysneeds.msg import audiodata
```

Comment:
what package is "audiosysneeds" ?? I've never heard of that.   I'm thinking that there needs to be a README file that describes what this code is, what prerequisites must be installed first, and then some simple examples and demos of how to use it, and what it's API is.